_posts/2020-11-08-distribute-policies-with-the-power-of-openfaas.md

```diff
@@ -0,0 +1,250 @@
+---
+title: "Distribute your policies with the power of OpenFaaS"
```

Comment:
> Distribute your policies with the power of OpenFaaS

Please think the problem being solved and write a problem-centric title.

For instance:

If I was writing about the IngressOperator, I might write: "Group your OpenFaaS functions into a single REST-style API"

Here perhaps?

"Share and distribute Open Policy Agent Bundles with functions"

Not every title needs to have the word openfaas in it, especially when it's on the openfaas domain.